stable/registry-creds-handler/values.yaml

```diff
@@ -0,0 +1,30 @@
+nameOverride:
```

Comment:
what i dont see here, is how you are going to pass the configuration needed for the authentication
something like

```yaml
# provide to allow application to create and refresh a docker registry auth secret
# registry:
#  secretName: blabla
#  namespace: default
#  kind:
#  auth:
#    username:
#    password:
```